docs/UserGuide.adoc

```diff
@@ -74,88 +74,100 @@ Examples:
 Shows a list of all persons in the address book. +
 Format: `list`
 
-=== Editing a person : `edit`
+=== Student Management
 
-Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+==== Enrolling a student in a class : `enroll`
+
+Enrolls a student in a class in the program. +
+
+Format: `enroll MATRIC_NUMBER CLASS_CODE MOD_CODE'
 
 ****
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index *must be a positive integer* 1, 2, 3, ...
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person's tags by typing `t/` without specifying any tags after it.
+* Enrolls the person with the given matriculation number in the class given by the class code of the module given by the module code.
```

Comment:
Maybe could be split into smaller dot points.
eg.
- Enrolls the person with the given matriculation number in the specified class
- The class should belong to the module with the MODULE_CODE
- The class should already in the program with the existing CLASS_CODE